File: php/URI/1117.php
<?php
$a;
$cont = 0;
$media = 0;
$j=0;

while ($cont < 2) {
    $a = trim(fgets(STDIN));
    if ($a >= 0.0 && $a <= 10.0) {
        $media = $media + $a;
        $cont++;
    } else print("nota invalida\n");

}
$media = $media / 2;
echo "media = " .number_format($media,2,".", "", )."\n";

?>

File: C/URI/1117.c
#include <stdio.h>
 
int main() {
 
  double a, media;
  int cont;
        cont = 0;
        media = 0;

        while(cont < 2)
        {

                scanf("%lf", &a);
                if(a >= 0 && a <= 10)
                {
                        media = media + a;
                        cont++;
                }
                else
                        printf("nota invalida\n");

        }
        media = media / 2;
        printf("media = %.2lf\n", media);



    return 0;
}